Increase space above headers to reduce overlap

mDuo13 opened the pull request:
Fixes #2520

mDuo13 committed:
Increase space above headers to reduce overlap

Fixes #2520

amarantha-k reviewed (commented):
While it addresses the issue, the extra space before the headings and especially page titles looks off.

<img width="893" alt="Screenshot 2024-04-11 at 10 22 12 AM" src="https://github.com/XRPLF/xrpl-dev-portal/assets/13425257/b0b16218-cf08-4925-bddb-f8a2318f5e9a">

If the issue noted in #2520 is a one-off or not seen in more than a handful of instances, should we reconsider changing the spacing?

mDuo13 committed:
Fix first h1 spacing, header font hierarchy

The font hierarchy is a little complicated between the _content.scss,
_font.scss, and Redocly theme styles, but this at least aligns the
Redocly theme variables with the MD content styling.

amarantha-k reviewed (approved):
The update to ensure that the page heading is not impacted looks good. Thanks for incorporating the feedback!
